governance/dashboard-update.py

```
@@ -0,0 +1,292 @@
+                              "state": sl[0][len("state:"):],
+                              "assignee": (it.get("assignees") or [{}])[0].get("login", ""),
+                              "url": it["html_url"], "updated_at": it.get("updated_at") or "",
+                              "days_idle": max(0, (NOW - _iso(it.get("updated_at"))).days)})
+            if len(batch) < 100:
+                break
+            page += 1
+    return cards
+
+
+def sli_automerge(repos):
+    """近 7 天 merged PR 中 App 身份合并占比（proxy；零分母→null N/A，#98 T2）。"""
```

Ruff (0.16.1)

[warning] Docstring contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF002)